config/advanced-install/namespaced-controller-wo-crds.yaml

```diff
@@ -190,8 +190,8 @@ data:
           metricsExporterImage: natsio/prometheus-nats-exporter:0.9.1
           configReloaderImage: natsio/nats-server-config-reloader:0.7.0
           startCommand: nats-server
-        - version: 2.9.4
-          natsImage: nats:2.9.4
+        - version: 2.9.6
```

Comment:
2.9.4 and 2.9.5 seem to have some bugs.

https://github.com/nats-io/nats-server/releases